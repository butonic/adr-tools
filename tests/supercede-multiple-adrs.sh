adr init docs madr
adr new First Record
adr new Second Record
adr new -s 2 -s 3 Third Record
head -10 docs/0002-first-record.md
head -10 docs/0003-second-record.md
tail -6 docs/0004-third-record.md
