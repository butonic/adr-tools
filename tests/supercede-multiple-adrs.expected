adr init docs madr
docs/0001-use-markdown-architectural-decision-records.md
adr new First Record
docs/0002-first-record.md
adr new Second Record
docs/0003-second-record.md
adr new -s 2 -s 3 Third Record
docs/0004-third-record.md
head -7 docs/0002-first-record.md
---
title: "2. First Record"
---

* Status: superseded by [4. Third Record]({{< ref "0004-third-record.md" >}})
* Deciders: [list everyone involved in the decision] <!-- optional -->
* Date: 1992-01-12
head -7 docs/0003-second-record.md
---
title: "3. Second Record"
---

* Status: superseded by [4. Third Record]({{< ref "0004-third-record.md" >}})
* Deciders: [list everyone involved in the decision] <!-- optional -->
* Date: 1992-01-12
tail -6 docs/0004-third-record.md
## Links <!-- optional -->

* [Link type] [Link to ADR] <!-- example: Refined by [ADR-0005](0005-example.md) -->
* â€¦ <!-- numbers of links can vary -->
* supersedes [2. First Record]({{< ref "0002-first-record.md" >}})
* supersedes [3. Second Record]({{< ref "0003-second-record.md" >}})
